return_dynamic_filters.php

```diff
@@ -135,19 +93,25 @@ function return_dynamic_filters_prepend_headers() {
 $t_function_name = 'print_filter_' . utf8_substr( $f_filter_target, 0, -7 ); # -7 for '_filter'
 if( function_exists( $t_function_name ) ) {
 	return_dynamic_filters_prepend_headers();
-	call_user_func( $t_function_name );
+	call_user_func( $t_function_name, $t_filter );
```

Comment:
This function is used to call function print_filter_do_filter_by_date( $p_hide_checkbox = false, array $p_filter = null )

==> call_user_func( $t_function_name, false, $t_filter ); 

Without it, no "use date filter" checkbox during date filter declaration and start and end dates are not init properly.